lib/crawly/middlewares/user_agent.ex

```diff
@@ -18,19 +18,25 @@ defmodule Crawly.Middlewares.UserAgent do
   """
   require Logger
 
-  def run(request, state, opts \\ []) do
+  def run(request, state, opts \\ %{}) do
```

Comment:
Elixir convention is for the default to be a keyword list

Reply:
@Ziinc I am addressing these lines: https://github.com/oltarasenko/crawly/blob/master/lib/crawly/middlewares/user_agent.ex#L26-L28

E.g. it has `Map.get(opts, :user_agents)` so opts has to be a map. But now after looking on the pipeline behavior definition I think you're right. I need to fix it in a different way. will fix